MsixCore/msixmgr/CommandLineInterface.cpp

```diff
@@ -479,4 +667,30 @@ void CommandLineInterface::SetWVDFileType(std::wstring fileType)
     {
         this->m_fileType = WVDFileType::NotSpecified;
     }
-}
\ No newline at end of file
+}
+
+std::wstring CommandLineInterface::GetFileTypeAsString()
+{
+    switch (this->m_fileType)
+    {
+        case WVDFileType::CIM: return L"CIM";
+        case WVDFileType::VHD: return L"VHD";
+        case WVDFileType::VHDX: return L"VHDX";
+        case WVDFileType::NotSpecified: return L"NotSpecified";
+    }
+}
+
+std::wstring CommandLineInterface::GetOperationTypeAsString()

```

Comment:
Where is this method getting used?

Reply:
This method is getting used in msixmgr.cpp file for telemetry logging, This method provides the String equivalent of the FileType